src/api/spec/factories/packages.rb

```diff
@@ -1,10 +1,12 @@
 FactoryGirl.define do
   factory :package do
+    project
     sequence(:name) { |n| "package_#{n}" }
     factory :package_with_file do
       after(:create) do |package|
         # NOTE: Enable global write through when writing new VCR cassetes.
         # ensure the backend knows the project
+CONFIG['global_write_through'] = true
```

Comment:
It's a leftover. I am currently trying to figure out why this http request is failing even though I created VCR cassettes for it.

I should have closed the PR for now. Sorry for the noise.